addons/resource/models/resource_calendar.py

```diff
@@ -105,7 +109,10 @@ def default_get(self, fields):
     two_weeks_explanation = fields.Char('Explanation', compute="_compute_two_weeks_explanation")
     flexible_hours = fields.Boolean(string="Flexible Hours",
                                     help="When enabled, it will allow employees to work flexibly, without relying on the company's working schedule (working hours).")
+    hours_per_week = fields.Float(compute="_compute_hours_per_week", string="Hours per Week", store=True)
```

Comment:
why do you want to store with compute ? Usually when we store a compute we want to be able to rewrite the values but here the value is still readonly :thinking:

Reply:
for me it's not necessary but keep it; at least the last reviewer will see yours reasons :D